applications/ContactStructuralMechanicsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -72,26 +71,6 @@ void  AddCustomUtilitiesToPython(pybind11::module& m)
     .def("InvertSearch",&TreeContactSearch<3, 4>::InvertSearch)
     ;
 
-    // Process Factory utility
-    class_<ProcessFactoryUtility, typename ProcessFactoryUtility::Pointer>(m, "ProcessFactoryUtility")
-    .def(init< >())
-    .def(init<  list&>())
-    .def(init<  object&>())
-    .def("AddProcess",&ProcessFactoryUtility::AddProcess)
-    .def("AddProcesses",&ProcessFactoryUtility::AddProcesses)
-    .def("ExecuteMethod",&ProcessFactoryUtility::ExecuteMethod)
-    .def("ExecuteInitialize",&ProcessFactoryUtility::ExecuteInitialize)
-    .def("ExecuteBeforeSolutionLoop",&ProcessFactoryUtility::ExecuteBeforeSolutionLoop)
-    .def("ExecuteInitializeSolutionStep",&ProcessFactoryUtility::ExecuteInitializeSolutionStep)
-    .def("ExecuteFinalizeSolutionStep",&ProcessFactoryUtility::ExecuteFinalizeSolutionStep)
-    .def("ExecuteBeforeOutputStep",&ProcessFactoryUtility::ExecuteBeforeOutputStep)
-    .def("ExecuteAfterOutputStep",&ProcessFactoryUtility::ExecuteAfterOutputStep)
-    .def("ExecuteFinalize",&ProcessFactoryUtility::ExecuteFinalize)
-    .def("IsOutputStep",&ProcessFactoryUtility::IsOutputStep)
-    .def("PrintOutput",&ProcessFactoryUtility::PrintOutput)
-    .def("Clear",&ProcessFactoryUtility::Clear)
-    ;
-
     // Sparse matrix multiplication utility
```

Comment:
I think this SpMM is a feature that should go in the core.

Reply:
SpMM?

Reply:
The sparse matrix utility I ask you and you said me to keep it on my application